src/main/scala/com/github/unchama/seichiassist/subsystems/seasonalevents/infrastructure/JdbcLastQuitPersistenceRepository.scala

```diff
@@ -1,21 +1,22 @@
 package com.github.unchama.seichiassist.subsystems.seasonalevents.infrastructure
 
-import java.time.LocalDateTime
-import java.util.UUID
-
-import com.github.unchama.seichiassist.subsystems.seasonalevents.domain.LastQuitPersistenceRepository
 import cats.effect.Sync
-import com.github.unchama.seichiassist.database.DatabaseConstants
+import com.github.unchama.seichiassist.subsystems.seasonalevents.domain.LastQuitPersistenceRepository
 import scalikejdbc.{DB, scalikejdbcSQLInterpolationImplicitDef}
 
+import java.time.LocalDateTime
+import java.time.format.DateTimeFormatter
+import java.util.UUID
+
 class JdbcLastQuitPersistenceRepository[F[_]](implicit SyncContext: Sync[F]) extends LastQuitPersistenceRepository[F, UUID] {
   override def loadPlayerLastQuit(key: UUID): F[Option[LocalDateTime]] = {
     SyncContext.delay {
       DB.localTx { implicit session =>
-        sql"select lastquit from ${DatabaseConstants.PLAYERDATA_TABLENAME} where uuid = '$key'"
-          .map { rs => rs.timestampOpt("lastquit").map(_.toLocalDateTime) }
-          .toList().apply().head
+        sql"select lastquit from playerdata where uuid = {uuid}".bindByName(Symbol("uuid") -> key.toString)
+          .map { rs =>
+            LocalDateTime.parse(rs.string("lastquit"), DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss.S"))
+          }.first().apply()
```

Comment:
nits: 単一であることを保障するためにsingleを使うべきだと思います

```suggestion
          }.single().apply()
```